cmd/download.go

```diff
@@ -123,21 +123,21 @@ func download(folder, release string) {
 		artifact := splittedArtifact[len(splittedArtifact)-1]
 		artifact = strings.Replace(artifact, ":", "_", 1)
 		cmd = exec.Command("skopeo", "copy", "docker://"+line, "dir://"+folder+"/"+artifact, "-q")
-		_, err := cmd.CombinedOutput()
+		stdout, err := cmd.CombinedOutput()
```

Comment:
Is there a reason to use cmd.CombinedOutput here, but executeCommand elsewhere in the function? Since executeCommand just calls cmd.CombinedOutput, it seems inconsistent to have a mix of these calls

Reply:
sure, I should have a generate command function for this logic, also so we can add unit tests.
OTOH is out of the scope for this PR, if you see the string(stdout) is all good, let's merge this, I will push another PR 
afterwards.